markdown/summary.Rmd

```diff
@@ -57,7 +56,7 @@ Note that hospital data and National Records of Scotland deaths data are provisi
 
 ### Main Points
 
-* In `r latest$fy`, there were `r latest$deaths` deaths in Scotland excluding those where an external cause of death was recorded (including falls).  For these individuals, `r latest$qom`% of their last six months of life was spent either at home or in a community setting, with the remaining `r latest$qom_hosp`% spent in hospital. This is equivalent to an average of `r round_half_up(182.5 * (latest$qom_hosp / 100))` days in hospital in the six months prior to death.
+* In `r latest$fy`, there were `r latest$deaths` deaths in Scotland excluding those where an external cause of death was recorded (including falls).  For these individuals, `r latest$qom`% of their last six months of life was spent either at home or in a community setting, with the remaining `r latest$qom_hosp`% spent in hospital. This is equivalent to an average of `r if(latest$hosp < 10){as.character(as.english(latest$hosp))}else{latest$hosp}` days in hospital in the six months prior to death.
 
 * Over the past `r as.character(as.english(year(end_date) - year(start_date)))` years, the percentage of time spent at home or in a community setting has gradually increased from `r summarise_data(basefile, include_years = "first") %>% pull(qom)`% in `r min(basefile$fy)` to `r latest$qom`% in `r latest$fy`.
 
```

Comment:
I can't comment for the background as it was approved. We've discussed it before. For this statement:

"The percentage of time spent at home or in a community setting towards the end of life provides a high level indication of progress in implementation of the national action plan."

Do you agree to change "national action plan" to "strategic framework objectives"? As strategic framework superseded the national action plan. I see the text is changed in \\nssstats01\irf\18-End-of-Life\Publication\EOL publication May 2018\2018-05-29-End-of-Life-Summary.docx

Reply:
Sorry Tina - I remember we discussed this and I never make the change - I agree and have changed in summary and report background sections.